packages/components/src/Card/Card.tsx

```diff
@@ -51,7 +51,7 @@ interface LinkCardProps extends CardProps {
 }
 
 interface ClickableCardProps extends CardProps {
-  onClick(): void;
+  onClick(e: React.MouseEvent<HTMLElement>): void;
```

Comment:
Still can't find a direct answer to how this mouse event _reacts_ on a mobile environment where there's no mouse

Reply:
Is there a way to test it? I don't know much about mobile.

Reply:
I think running atlantis on your phone should do the trick

Reply:
works fine, soooo... 🤷‍♂